pootle/static/js/editor/app.js

```diff
@@ -2312,4 +2338,40 @@ PTL.editor = {
     return true;
   },
 
+  toggleSuggestion(e, canHide = false) {
+    e.stopPropagation();
+    const $suggestion = $(e.target).parents('.js-user-suggestion');
```

Comment:
Note how error-prone this is: any change in layout can potentially break functionality.

If I'm not mistaken `e.dataset.suggId` already contains the suggestion id, therefore the DOM node can be retrieved as follows:
```js
const suggestion = document.getElementById(`suggestion-${e.dataset.suggId}`);
```

Along with this you'll need to adjust below the jQuery APIs by DOM ones (`.data()` => `.dataset.`, `.addClass()` => `.classList.add()`)